scandium-core/src/main/java/org/eclipse/californium/scandium/util/PskUtil.java

```diff
@@ -36,29 +36,27 @@
 	
 	private PreSharedKeyIdentity pskIdentity;
 	
-	/**
-	 * Support Server Name Indication TLS extension.
-	 */
-	protected boolean sniEnabled = true;
-	
 	/**
 	 * Retrieves preshared key identity and preshared key for the given dtls session and psk store.
 	 * 
+	 * @param isSniEnabled - true if current {@code DTLSSession} supports 
```

Comment:
this is not true. this flag is explicitly set by the user and indicates whether the DtlsConnector supports SNI *at all*.